README.md

```diff
@@ -116,7 +116,33 @@ Create ConfigMap for Kafka Connect Log4j configuration:
 
 1. Update the namespace in `kafka-connect.yaml`
 1. `kubectl -n <namespace> apply -f kafka-connect.yaml`
-1. `curl <serviceIP>:<servicePort>/connector-plugins` to see the MQ Source connector available to use
+1. `curl <serviceIP>:<servicePort>/connector-plugins` to see whether the MQ source connector is available to use
+
+### Deploying to OpenShift using Strimzi
+
+This repository includes a Kubernetes yaml file called `strimzi.kafkaconnect.yaml` for use with the [Strimzi](https://strimzi.io) operator. Strimzi provides a simplified way of running the Kafka Connect distributed worker, by defining either a KafkaConnect resource or a KafkaConnectS2I resource.
+
+The KafkaConnectS2I resource provides a nice way to have OpenShift do all the work of building the Docker images for you. This works particularly nicely combined with the KafkaConnector resource that represents an individual connector.
+
+The following instructions assume you are running on OpenShift and have Strimzi 0.16 or later installed.
+
+#### Start a Kafka Connect cluster using KafkaConnectS2I
+1. Create a file called `kafkaconnect.yaml` containing the definition of a KafkaConnectS2I resource. Configure it with information it needs to connect to your Kafka cluster. You must include the annotation `strimzi.io/use-connector-resources: "true"` to configure it to use KafkaConnector resources so you can avoid needing to call the Kafka Connect REST API directly.
```

Comment:
Is it worth recommending that developers look in the examples folder of their Strimzi download to find an example? Also, I wonder if we should match the examples folder naming convention which is  `kafka-connect-s2i.yaml` to reduce the likelyhood that the wrong file is chosen.